Makefile

```diff
@@ -171,5 +171,5 @@ fsm_graph: clean
 .PHONY: clean
 clean:
 	@echo "cleaning up caches and output"
-	go clean -cache -testcache -r -x ./... 2>&1 >/dev/null
+	go clean -cache -testcache -r
```

Comment:
We might want to keep the `/dev/null` redirect, this command can generate a lot of output which doesn't seem to be useful.